arches/app/src/arches/themes/default.ts

```diff
@@ -0,0 +1,63 @@
+import { definePreset } from "@primevue/themes";
+import Aura from "@primevue/themes/aura";
+
+export const ArchesPreset = definePreset(Aura, {
```

Comment:
I'd like this first iteration to be intentional, more of a blueprint for what's getting fleshed out in further iterations. 

I like the idea of `arches` primitives, but I'm unsure of supporting legacy coloring, or at very least without a `-legacy` tag. Ideally moving forward there will be a small subset of colors that define an out-of-the-box Arches application. On that, the color gradation of `arches.blue` seems to buck the logic of what PrimeVue's doing with their other colors: moving up/down the gradient scale. 

Although I like `arches` primitives, maybe it's _not_ the right solution? Maybe our solution is just all semantic tokens: `arches.navigation.blue`, `arches.background`, `arches.button.red` etc etc. 

Should we have a `primary` semantic color that's just iterating the `sky` gradient? IMO I'd say pull that out.

The component overrides, do you think these are generic enough to be universal? Would we ever want a datatable or button label without the added fontWeight, for instance?

`darkModeSelector` should be related to a function that takes in user interaction with the theming selector. Doesn't need to be done in this PR but it's a case to handle around

Reply:
Agree on the legacy namespace.

> On that, the color gradation of arches.blue seems to buck the logic of what PrimeVue's doing with their other colors: moving up/down the gradient scale.

Right, my thought also. When we do have that 6-8 color subset, hopefully we can draw from one of those gradients.

Reply:
> Should we have a primary semantic color that's just iterating the sky gradient? IMO I'd say pull that out.

Aura uses emerald (green) as its primary color. I'm happy to remove it, I just thought that arches had mostly blue as its primary color, so I wanted to repoint to a blue shade.

Reply:
> The component overrides, do you think these are generic enough to be universal? Would we ever want a datatable or button label without the added fontWeight, for instance?

Let's include folks who care about design details in this specific conversation, but yes, the only reason these are here are to enforce it sitewide (obviously subject to further undoing at the component level case by case).

The splitter handle override is definitely universal, if I had the time I'd report it to primevue as a bug. I bet someone fixes it eventually and we can back it out.

Reply:
> Maybe our solution is just all semantic tokens: arches.navigation.blue, arches.background, arches.button.red etc etc.

I like that. And if we go that route, then having the legacy tokens as primitives makes sense, because their context is not relevant now, they're not swappable, they're frozen in time.

Reply:
With the caveat that I'm kind of hoping arches.navigation.blue resolves to sky.800 if possible rather than a raw value of our own :D

Reply:
> Let's include folks who care about design details in this specific conversation, but yes, the only reason these are here are to enforce it sitewide (obviously subject to further undoing at the component level case by case).
> 
> The splitter handle override is definitely universal, if I had the time I'd report it to primevue as a bug. I bet someone fixes it eventually and we can back it out.

Agreed that we should include other people in this conversation 👍 . That said, my kneejerk reaction here is to keep universal overrides at a minimum.

Reply:
I'll pull it out -- I'm wondering if this is because something else was going awry with font sizes in the controlled list manager (w/r/t arches chrome and such).

The extra weight looks a little silly in lingo.

Reply:
Going with a lighter touch, so it's gone again